### PR TITLE
Add DRAWING style

### PR DESCRIPTION
**Pull request by Sven-J-Steinert**
closed lines in console

```
│ City name │ Area │ Population │ Annual Rainfall │
├───────────┼──────┼────────────┼─────────────────┤
│ Brisbane  │ 5905 │    1857594 │          1146.4 │
│ Darwin    │  112 │     120900 │          1714.7 │
│ Hobart    │ 1357 │     205556 │           619.5 │
```

### CODE REVIEW

File: src/prettytable/prettytable.py
Comment (hugovk) [inline src/prettytable/prettytable.py]:
```suggestion
DRAWING = 16
```

https://github.com/jazzband/prettytable/pull/104 will be 15, so let's go for 16.

Please could you also add a test case so the new code is covered?

**Review by hugovk — CHANGES_REQUESTED** [review]

File: src/prettytable/prettytable.py
Comment (Sven-J-Steinert) [reply src/prettytable/prettytable.py]: Sure, i've done the changes and run the test - should work now

File: src/prettytable/prettytable.py
Comment (hugovk) [inline src/prettytable/prettytable.py:1311]: Can this be done without introducing a new variable just for this style?

https://github.com/jazzband/prettytable/pull/104 will introduce a bunch of new junction chars, will one of them help here?

Comment (Sven-J-Steinert) [reply src/prettytable/prettytable.py:1311]: yes sure, i can use left_junction_char and right_junction_char but how can i include this usage if #104 isn't merged yet?
Doenst it cause conflicts? Im sorry, im new to this workflow

Comment (hugovk) [reply src/prettytable/prettytable.py:1311]: Good questions! We can either wait until #104 is merged, or hoist bits of that code into here.

I think it's better to wait, hopefully #104 will be merged very soon, it's almost ready.

There will probably be merge conflicts whichever is merged first. That one's a bigger and older PR so makes sense to get that in first, will be easier to sort out the conflicts in this smaller one.

Don't worry, I can help with that if needed.

Thank you!

Comment (hugovk) [reply src/prettytable/prettytable.py:1311]: #104 has been merged.

Comment (hugovk) [reply src/prettytable/prettytable.py:1311]: I've resolved the conflicts in this PR after the merge of #104.

Do you think you can update this now to replace `self.drawing`?

Comment (Sven-J-Steinert) [reply src/prettytable/prettytable.py:1311]: Yes i would like to, but i am not sure if i should bring my fork to the newest version of the master first? or can i edit it directly here? i want to avoid merge conflicts

Comment (hugovk) [reply src/prettytable/prettytable.py:1311]: Yes, you should update your fork's branch to match this. Editing directly here is possible too and may be easier, click the pencil edit icon at the top right here:

https://github.com/unconsciou5/prettytable/blob/master/src/prettytable/__init__.py

Comment (Sven-J-Steinert) [reply src/prettytable/prettytable.py:1311]: okay, maybe i just refork it and open a new pull request, so i can run the tests too with the new functions

Comment (Sven-J-Steinert) [reply src/prettytable/prettytable.py:1311]: wait, i just saw the style is already implemented as "SINGLE_BORDER" this makes the DRAWING one obsolete.

Comment (hugovk) [reply src/prettytable/prettytable.py:1311]: They're slightly different, but if `SINGLE_BORDER` suits your purposes then that's good to hear!

`SINGLE_BORDER`

```
┌─────────┬─────────┬─────────┐
│ Field 1 │ Field 2 │ Field 3 │
├─────────┼─────────┼─────────┤
│ value 1 │  value2 │  value3 │
│ value 4 │  value5 │  value6 │
│ value 7 │  value8 │  value9 │
└─────────┴─────────┴─────────┘
```

`DRAWING`

```
│ Field 1 │ Field 2 │ Field 3 │
├─────────┼─────────┼─────────┤
│ value 1 │  value2 │  value3 │
│ value 4 │  value5 │  value6 │
│ value 7 │  value8 │  value9 │
```

Anyway, thank you for your contribution!

**Pull request closed without merging**
